src/main/java/com/taboola/backstage/model/Hour.java

```diff
@@ -0,0 +1,62 @@
+package com.taboola.backstage.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Created by vladi.m
+ * Date 03/10/2018
+ * Time 16:22
+ * Copyright Taboola
+ */
+public enum Hour {
```

Comment:
why did you overcomplicated it ?
why not representing with  integer and validate its between [0,24] ?

Reply:
I could but then it would be runtime error and not compile time error.